notebooks/Learning_Rate_Scheduler.ipynb

```diff
@@ -0,0 +1,3350 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Learning Rate Schedulers with Skorch \n",
+    "\n",
+    "This notebook demonstrates 3 learning rate schedulers in skorch:\n",
+    "\n",
+    "StepLR, ReduceROn Plateau and Cosine Annealing. This notebook was contributed by [Parag Ekbote](https://github.com/ParagEkbote).\n",
+    "\n",
+    "Firstly you will need to install the following libraries: skorch,numpy,matplotlib and torch."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a href=\"\" target=\"_blank\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open in Colab\"></a>\n"
```

Comment:
The colab link is not correct, as the href is missing, right?

Reply:
This a minor nit based off our previous approach. Updated as discussed.